Cargo.toml

```diff
@@ -74,6 +73,9 @@ xterm-query = { version = "0.5", optional = true }
 [dev-dependencies]
 glassbench = "0.4.4"
 
+[target.'cfg(any(target_os = "windows", all(unix, not(any(target_os = "macos", target_os = "ios", target_os = "android")))))'.dependencies]
```

Comment:
But the trash feature compiles and works on MacOS

Reply:
Oh ok, I had just copied the same conditional guard that was used here: 

https://github.com/Canop/broot/blob/3de3c2e44c536df645a8b1b81935b611ad667577/src/verb/verb_store.rs#L173-L181

Should these extra trash commands be available on MacOS too?